docs/sources/tempo/configuration/grafana-agent/tail-based-sampling.md

```diff
@@ -50,194 +57,142 @@ If you're using a multi-instance deployment of the agent,
 add load balancing and specify the resolving mechanism to find other Agents in the setup.
 To see all the available configuration options, refer to the [configuration reference](/docs/agent/latest/configuration/traces-config/).
 
-```yaml
-traces:
-  configs:
-    - name: default
-      ...
-      tail_sampling:
-        policies:
-          # sample traces that have a total duration longer than 100ms
-          - latency:
-              threshold_ms: 100
-          # sample traces that contain at least one span with status code ERROR
-          - status_code:
-              status_codes:
-                - "ERROR"
-      load_balancing:
-        resolver:
-          dns:
-            hostname: host.namespace.svc.cluster.local
+## Example for Grafana Agent Flow
+
+[Grafana Agent Flow](/docs/agent/latest/flow/) is a component-based revision of Grafana Agent with a focus on ease-of-use, debuggability, and ability to adapt to the needs of power users.
+Flow configuration files written in River instead of YAML.
+
+Grafana Agent Flow uses the [`otelcol.processor.tail_sampling component`](/docs/agent/latest/flow/reference/components/otelcol.processor.tail_sampling/)` for tail-based sampling.
+
+```river
+otelcol.receiver.otlp "otlp_receiver" {
+    grpc {
+        endpoint = "0.0.0.0:4317"
+    }
+
+    output {
+        traces = [
+            otelcol.processor.tail_sampling.errors.input,
+        ]
+    }
+}
+
+otelcol.exporter.otlp "tempo" {
+    client {
+        endpoint = "tempo:4317"
+    }
+}
+
+// The Tail Sampling processor will use a set of policies to determine which received traces to keep
+// and send to Tempo.
+otelcol.processor.tail_sampling "errors" {
+    // Total wait time from the start of a trace before making a sampling decision. Note that smaller time
+    // periods can potentially cause a decision to be made before the end of a trace has occurred.
+    decision_wait = "30s"
+    // The number of traces to keep in memory by the Agent.
```

Comment:
Get rid of lines 92 and 93. There's actually something that can go wrong here if there's a high latency.